### ISSUE TITLE
Chunk Save Error / Armor stands disappearing & item frame items disappearing 

### ISSUE THREAD

**Issue by MagmaTeam** [IzzelAliz/Arclight#744]
### I have confirmed that ...

- [X] Arclight is up to date
- [X] all dependencies are installed
- [X] all plugins and mods are up to date
- [X] unable to reproduce in Spigot
- [X] unable to reproduce in Forge

### Arclight version

1.16.5-1.0.24-SNAPSHOT

### OS & Java versions

Java 11

### Plugins and Mods

```raw
[18:17:54] [Server thread/INFO] [Console/]: ------------=[ ServerUtils Plugins ]=-------------
[18:17:54] [Server thread/INFO] [Console/]:  Plugins (37): BetterRTP, BuycraftX, CoralStay, CoreProtect, DeluxeMenus, DiscordSRV, Essentials, EssentialsChat, EssentialsDiscord, EssentialsEconomyBridge, EssentialsSpawn, FastAsyncWorldEdit, FastChunkPregenerator, GriefDefender, HolographicDisplays, InteractiveBoard, InteractiveChat, LiteBans, LuckPerms, MarriageMaster, Multiverse-Core, MyCommand, PVPToggle, PlaceholderAPI, PlayerKits, ProtocolLib, RewardsLite, ServerUtils, ShopGUIPlus, ShopGUIPlus-SellGUI, SimpleVoteListener, TabList, Vault, VoidGenerator, Votifier, WorldBorder and spark
```

### Description

Having this chunk save error, https://mcpaste.io/c394ad8060d7547b

Could also be apart of the reason why sense the update, my armor stands keep deleting as well as what is in item frames. 

### Step to reproduce

basically is just always spamming console

### Logs

https://mcpaste.io/c394ad8060d7547b
